tardis/io/model_reader.py

```diff
@@ -256,3 +264,31 @@ def read_simple_ascii_abundances(fname):
     abundances = pd.DataFrame(data[1:,1:].transpose(), index=np.arange(1, data.shape[1]))
 
     return index, abundances
+
+
+def read_simple_isotope_abundances(fname, delimiter=','):
```

Comment:
the default delimiter should be a whitespace. this is a special option in `read_csv`.